example.php

```diff
@@ -48,436 +112,257 @@ function getSSLPage($url) {
-        ])
-        ->setExclude([
-            'services' => [
-                ['name' => 'assistant'],
-                ['name' => 'avatars'],
+        $sdk  = new SDK($language, new Swagger2($spec));
+
+        configureSDK($sdk, [
+            'version' => '0.16.0',
+            'url' => 'https://appwrite.io',
+            'license' => 'BSD-3-Clause',
+            'gitUserName' => 'appwrite',
+            'gitRepoName' => 'sdk-for-cli',
+            'defaultHeaders' => ['X-Appwrite-Response-Format' => '1.7.0'],
+            'exclude' => [
+                'services' => [
+                    ['name' => 'assistant'],
+                    ['name' => 'avatars'],
+                ],
             ],
-        ])
-    ;
+        ]);
 
-    $sdk->generate(__DIR__ . '/examples/cli');
+        $sdk->generate(__DIR__ . '/examples/cli');
+    }
 
     // Ruby
-    $sdk  = new SDK(new Ruby(), new Swagger2($spec));
-
-    $sdk
-        ->setName('NAME')
-        ->setDescription('Repo description goes here')
-        ->setShortDescription('Repo short description goes here')
-        ->setURL('https://example.com')
-        ->setLogo('https://appwrite.io/v1/images/console.png')
-        ->setLicenseContent('test test test')
-        ->setWarning('**WORK IN PROGRESS - NOT READY FOR USAGE**')
-        ->setChangelog('**CHANGELOG**')
-        ->setGitUserName('repoowner')
-        ->setGitRepoName('reponame')
-        ->setTwitter('appwrite_io')
-        ->setDiscord('564160730845151244', 'https://appwrite.io/discord')
-        ->setDefaultHeaders([
-            'X-Appwrite-Response-Format' => '1.6.0',
-        ])
-    ;
-
-    $sdk->generate(__DIR__ . '/examples/ruby');
+    if (!$requestedSdk || $requestedSdk === 'ruby') {
+        $sdk  = new SDK(new Ruby(), new Swagger2($spec));
+
+        configureSDK($sdk);
+
+        $sdk->generate(__DIR__ . '/examples/ruby');
+    }
 
     // Python
-    $sdk  = new SDK(new Python(), new Swagger2($spec));
-
-    $sdk
-        ->setName('NAME')
-        ->setVersion('7.2.0')
-        ->setDescription('Repo description goes here')
-        ->setShortDescription('Repo short description goes here')
-        ->setURL('https://example.com')
-        ->setLogo('https://appwrite.io/v1/images/console.png')
-        ->setLicenseContent('test test test')
-        ->setWarning('**WORK IN PROGRESS - NOT READY FOR USAGE**')
-        ->setChangelog('**CHANGELOG**')
-        ->setGitUserName('repoowner')
-        ->setGitRepoName('reponame')
-        ->setTwitter('appwrite_io')
-        ->setDiscord('564160730845151244', 'https://appwrite.io/discord')
-        ->setDefaultHeaders([
-            'X-Appwrite-Response-Format' => '1.6.0',
-        ])
-    ;
-
-    $sdk->generate(__DIR__ . '/examples/python');
+    if (!$requestedSdk || $requestedSdk === 'python') {
+        $sdk  = new SDK(new Python(), new Swagger2($spec));
+
+        configureSDK($sdk, [
+            'version' => '7.2.0',
```

Comment:
i think its fine to have a static version for all as this is just a test/example script.